content/zh-cn/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -1,9 +1,21 @@
 ---
-<!--title: Using a Service to Expose Your App-->
 title: 使用 Service 暴露你的应用
 weight: 10
+description: |-
+    学习关于 Kubernetes 中的 Service.
+    理解 Label 和 Selector 如何关联到 Service.
+    暴露一个应用程序到 Kubernetes 集群外部。
```

Comment:
```suggestion
    在 Kubernetes 集群外暴露应用。
```
line 43 has the similar expression. For consistency.